frontend/src/components/landing/BundleBanner.tsx

```diff
@@ -0,0 +1,134 @@
+import { Localized, useLocalization } from "@fluent/react";
+import { getBundlePrice } from "../../functions/getPlan";
+import { RuntimeData } from "../../hooks/api/runtimeData";
+import { Button } from "../Button";
+import { MozillaVpnWordmark } from "../Icons";
+import styles from "./BundleBanner.module.scss";
+import womanInBanner from "./images/bundle-banner-woman.png";
+import bundleFloatOne from "./images/bundle-float-1.svg";
+import bundleFloatTwo from "./images/bundle-float-2.svg";
+import bundleFloatThree from "./images/bundle-float-3.svg";
+import bundleLogo from "./images/vpn-and-relay-logo.svg";
+
+type FloatingFeaturesProps = {
+  icon: string;
+  text: string;
+  position: string;
+};
+
+export type Props = {
+  runtimeData: RuntimeData;
+};
+
+const FloatingFeatures = (props: FloatingFeaturesProps) => {
+  return (
+    <div
+      className={`${styles[props.position]} ${styles["float-features-item"]}`}
+    >
+      <img alt={props.text} src={props.icon} />
+      <span className={styles["float-features-text"]}>{props.text}</span>
+    </div>
+  );
+};
+
+export const BundleBanner = (props: Props) => {
+  const { l10n } = useLocalization();
+
+  const mainImage = (
+    <img
+      src={womanInBanner.src}
```

Comment:
1.15MB is a whole lot of data to add to the page weight, especially on metered connections. I haven't used it before, but I think `srcset` can help serving relevant images responsively; see https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images.

And if you want to make sure you use the same media queries in the `sizes` attribute as we do in our styles, I think you should be able to import them [from `mediaQuery.module.scss`](https://github.com/mozilla/fx-private-relay/blob/main/frontend/src/hooks/mediaQuery.module.scss) as `import { mq_xs, mq_sm, mq_md, mq_lg, mq_xl } from "../../hooks/mediaQuery.module.scss";` (example [here](https://github.com/mozilla/fx-private-relay/blob/584273c50f10a1c09363d26d69874bc99ad229a1/frontend/src/hooks/mediaQuery.ts#L2)).